src/http/server.ts

```diff
@@ -53,6 +53,12 @@ export class Server {
    */
   public port: number = 1447;
 
+  /**
+   * A property to hold all paths associated with their resources for lookups
+   * during the request-resource lifecycle.
+   */
+  protected paths: any = {};
```

Comment:
Can't we strongly type this? If not, please change it to `unknown`. Let's start avoiding `any`.

Reply:
43bb3a4